test-utils/utils/utils.go

```diff
@@ -96,10 +96,9 @@ func (u *Utils) GetGCSListURL() string {
 	return fmt.Sprintf(GCSListAPIURLTemplate, u.bucket)
 }
 
-// GetPathToJenkinsGoogleBucket reads data from Google project's GCS bucket for the given job and buildNumber.
-// Returns a response with file stored under a given (relative) path or an error.
-func (u *Utils) GetPathToJenkinsGoogleBucket(job string, buildNumber int, path string) string {
-	return fmt.Sprintf("%v/%v/%v/%v", u.GetGCSDirectoryURL(), job, buildNumber, path)
+// GetPathToJenkinsGoogleBucket returns a GCS path containing the artifacts for a given job and buildNumber.
```

Comment:
Please fix this comment. Note that it only formats a path, and doesn't include a host.